restlicsharpclient/restliclient/request/GetRequest.cs

```diff
@@ -15,24 +15,40 @@
 */
 
 using System.Collections.Generic;
+using System;
 
 using restlicsharpdata.restlidata;
+using restlicsharpclient.restliclient.response;
+using restlicsharpclient.restliclient.response.decoder;
+using restlicsharpclient.restliclient.util;
 
 namespace restlicsharpclient.restliclient.request
 {
-    public class GetRequest<TKey, TEntity> : Request where TEntity : RecordTemplate
+    /// <summary>
+    /// Highest abstracted representation of a rest.li GET request.
```

Comment:
Remove those `Highest abstracted` from your top-level class summary, give people confusion that this is an abstract class, it is not. `Representation of a Rest.li GET request` is pretty fine. This comment is also applicable for other class comments.